templates/webapi/test/settings.html.ep

```diff
@@ -4,9 +4,17 @@
     </thead>
     <tbody>
         % my $s = $job->settings_hash;
+        % my @links = split ",", app->config->{job_settings_ui}->{keys_to_render_as_links};
         % for my $k (sort keys %$s) {
-            % my $v = $s->{$k};
-            <tr><td><%= $k %></td><td><%= $v =~ m{^https?://} ? link_to $v => $v : $v %></td></tr>
+          % my $v = $s->{$k};
+          <tr><td><%= $k %></td>
+          % if (@links && link_key_exists($k, \@links)) {
+            <td><%=  link_to $v => $job->id . "/settings/" . $v %></td>
+          % }
+          % else {
+            <td><%= $v =~ m{^https?://} ? link_to $v => $v : $v %></td>
+          % }
+          </tr>
```

Comment:
I would still like to keep this a one-liner in the template and make the table cell contents via a helper.

Reply:
Changed made. I removed code that it does not belong in the template in the helper and i just append to condition a check for the keys.